### PR TITLE
Add account based support via CFN

### PR DESCRIPTION
**Pull request by shubham-d**
*Issue #, if available:*
Customers can create account level application using CLoudformation

*Description of changes:*
Support adding account based support via CFN

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-applicationinsights-application/src/main/java/software/amazon/applicationinsights/application/HandlerHelper.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
 import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
 import software.amazon.cloudformation.proxy.Logger;
 import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import sun.java2d.loops.CustomComponent;
```
Comment (noahrin) [inline aws-applicationinsights-application/src/main/java/software/amazon/applicationinsights/application/HandlerHelper.java:42]: This doesn't look right

File: aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/CreateHandlerTest.java [hunk 2]
```diff
@@ -12,43 +14,99 @@
...
-//
-//        assertThat(response).isNotNull();
-//        assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
-//        assertThat(response.getCallbackContext()).isNull();
-//        assertThat(response.getCallbackDelaySeconds()).isEqualTo(0);
-//        assertThat(response.getResourceModel()).isEqualTo(request.getDesiredResourceState());
-//        assertThat(response.getResourceModels()).isNull();
-//        assertThat(response.getMessage()).isNull();
-//        assertThat(response.getErrorCode()).isNull();
-//    }
```
Comment (noahrin) [inline aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/CreateHandlerTest.java:53]: Should this be uncommented?

File: aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/DeleteHandlerTest.java [hunk 3]
```diff
@@ -12,43 +14,82 @@
...
-//
-//        assertThat(response).isNotNull();
-//        assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
-//        assertThat(response.getCallbackContext()).isNull();
-//        assertThat(response.getCallbackDelaySeconds()).isEqualTo(0);
-//        assertThat(response.getResourceModel()).isEqualTo(request.getDesiredResourceState());
-//        assertThat(response.getResourceModels()).isNull();
-//        assertThat(response.getMessage()).isNull();
-//        assertThat(response.getErrorCode()).isNull();
-//    }
```
Comment (noahrin) [inline aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/DeleteHandlerTest.java:53]: Uncomment or remove?

File: aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/UpdateHandlerTest.java [hunk 4]
```diff
@@ -12,43 +14,106 @@
...
-//
-//        assertThat(response).isNotNull();
-//        assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
-//        assertThat(response.getCallbackContext()).isNull();
-//        assertThat(response.getCallbackDelaySeconds()).isEqualTo(0);
-//        assertThat(response.getResourceModel()).isEqualTo(request.getDesiredResourceState());
-//        assertThat(response.getResourceModels()).isNull();
-//        assertThat(response.getMessage()).isNull();
-//        assertThat(response.getErrorCode()).isNull();
-//    }
```
Comment (noahrin) [inline aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/UpdateHandlerTest.java:53]: Uncomment or remove?

**Review by noahrin — CHANGES_REQUESTED** [review]

File: aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/UpdateHandlerTest.java (hunk 4 above)
Comment (shubham-d) [reply aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/UpdateHandlerTest.java:53]: Removed the commented code

File: aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/DeleteHandlerTest.java (hunk 3 above)
Comment (shubham-d) [reply aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/DeleteHandlerTest.java:53]: Removed the commented code

File: aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/CreateHandlerTest.java (hunk 2 above)
Comment (shubham-d) [reply aws-applicationinsights-application/src/test/java/software/amazon/applicationinsights/application/CreateHandlerTest.java:53]: Removed the commented code

File: aws-applicationinsights-application/src/main/java/software/amazon/applicationinsights/application/HandlerHelper.java (hunk 1 above)
Comment (shubham-d) [reply aws-applicationinsights-application/src/main/java/software/amazon/applicationinsights/application/HandlerHelper.java:42]: Removing the import statement

**Pull request closed without merging**